exporter/opentelemetry-exporter-zipkin-json/tests/encoder/test_v2_json.py

```diff
@@ -203,3 +206,11 @@ def _test_encode_max_tag_length(self, max_tag_value_length: int):
                 [otel_span], NodeEndpoint()
             ),
         )
+
+    def test_dropped(self):
+        otel_span = get_span_with_dropped_attributes_events_links()
+        tags = JsonV2Encoder()._encode_span(otel_span, "test")["tags"]
+
+        self.assertEqual("1", tags.get("otel.dropped_links_count"))
```

Comment:
won't the result be the same that the test would fail though?

Reply:
It will fail, but with an `AssertionError` if the value is not as expected but with a `KeyError` if the key is not present. This different exception can make it easier to understand what went wrong.

Reply:
Fair enough, updated.